src/lava/magma/runtime/message_infrastructure/message_infrastructure/csrc/message_infrastructure_py_wrapper.cc

```diff
@@ -161,6 +161,11 @@ PYBIND11_MODULE(MessageInfrastructurePywrapper, m) {
     .def("service", &CppVarPortScalarSparse::Service)
     .def("recv", &CppVarPortScalarSparse::Recv)
     .def("peek", &CppVarPortScalarSparse::Peek);
+  py::class_<IdentityTransformer> (m, "IdentityTransformer")
```

Comment:
We need to change the transformer logic. The python side transfer a function to achieve the transformer